EIPS/eip-txindex-opcode.md

```diff
@@ -0,0 +1,48 @@
+---
+title: TXINDEX opcode
+description: Opcode to get index of transaction within block
+author: Marc Harvey-Hill (@Marchhill), Ahmad Bitar (@smartprogrammer93)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in EthMagicians: https://ethereum-magicians.org/c/eips/5